include/shell/shell_bt_nus.rst

```diff
@@ -13,66 +13,74 @@ Setup
 
 Enable NUS shell transport in your application to be able to receive shell commands remotely.
 
-To send shell commands to an application that uses this module, you need specific software, comparable to a terminal (for example, PuTTY) for UART communication.
+To send shell commands to an application that uses this module, you need a serial console application, like PuTTY, for UART communication.
```

Comment:
actually, this requirement applies only to `bt_nus_shell.py` (not to `BLE Console`) and it is mentioned in testing section so this sentence might be removed from here.

Reply:
fixed